src/Microsoft.DotNet.Cli.Utils/Constants.cs

```diff
@@ -20,6 +20,7 @@ internal static class Constants
 
         public static readonly string HostExecutableName = "corehost" + ExeSuffix;
         public static readonly string DefaultConfiguration = "Debug";
+        public static readonly string DefaultPublishConfiguration = "Release";
```

Comment:
First one I think is used by "Compile" and "Run" where it makes sense to use Debug as you are iterating with testing your app manually. I believe every single person who publishes their app uses Release configuration as it is just faster and you don't want users to see any debug data. I actually think that we shouldn't have --configuration switch in here although not pushing on that since it sometimes may come useful. (Release IMO should still produce pdbs so this is just about optimizations)